src/services

```diff
@@ -24,7 +24,7 @@ p({"class":"foam.nanos.boot.NSpec", "name":"logLevelFilterLogger",             "
 p({"class":"foam.nanos.boot.NSpec", "name":"logger",                           "lazy":false, "serve":false, "serviceScript":"import foam.nanos.logger.CompositeLogger; import foam.nanos.logger.StdoutLogger; import foam.nanos.logger.FileLogger; import foam.nanos.logger.ProxyLogger; import foam.nanos.logger.LogLevelFilterLogger; import foam.nanos.logger.DAOLogger; CompositeLogger log = new CompositeLogger(); StdoutLogger stdOut = new StdoutLogger(); ProxyLogger logLevelFilterLogger = (ProxyLogger) x.get(\"logLevelFilterLogger\"); logLevelFilterLogger.setX(x); logLevelFilterLogger.setDelegate(log); DAOLogger daoLogger = new DAOLogger(x); log.add(stdOut).add(daoLogger); return logLevelFilterLogger;"})
 
 p({"class":"foam.nanos.boot.NSpec", "name":"localNotificationDAO", "lazy":false, "serve":false, "serviceScript":"return  new foam.dao.EasyDAO.Builder(x).setOf(foam.nanos.notification.Notification.getOwnClassInfo()).setIndex(new foam.core.PropertyInfo[] {foam.nanos.notification.Notification.GROUP_ID}).setAuthenticate(false).setPm(true).setSeqNo(true).setDecorator(new foam.nanos.notification.NotificationTemplateDAO.Builder(x).setDelegate(new foam.nanos.notification.SendNotificationDAO(x, new foam.nanos.notification.SendEmailNotificationDAO(new foam.nanos.notification.SendSlackNotificationDAO(x)))).build()).setJournalType(foam.dao.JournalType.SINGLE_JOURNAL).setJournalName(\"notifications\").build();"})
-p({"class":"foam.nanos.boot.NSpec", "name":"notificationDAO",                  "lazy":false,  "serve":true,  "serviceScript":"return new foam.dao.EasyDAO.Builder(x).setOf(foam.nanos.notification.Notification.getOwnClassInfo()).setAuthorize(true).setInnerDAO((foam.dao.DAO) x.get(\"localNotificationDAO\")).build();", "client":"{\"of\":\"foam.nanos.notification.Notification\"}"})
+p({"class":"foam.nanos.boot.NSpec", "name":"notificationDAO",                  "lazy":false,  "serve":true,  "serviceScript":"return new foam.dao.EasyDAO.Builder(x).setOf(foam.nanos.notification.Notification.getOwnClassInfo()).setAuthorize(true).setAuthenticate(false).setInnerDAO((foam.dao.DAO) x.get(\"localNotificationDAO\")).build();", "client":"{\"of\":\"foam.nanos.notification.Notification\"}"})
```

Comment:
I don't think this is the best fix, this will allow all notifications to be viewed freely by anyone.

Reply:
Authorization checks for permissions

Reply:
oh shows how much I know. @penzital and thx @noodlemoodle 

I didn't realize that there were two: Authorize and Authenticate.

Sorry guys.